.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       matrix:
         include:
           - elixir: 1.9.4
-            otp: 20.3.8.26
+            otp: 21.2.7
```

Comment:
Since https://github.com/phoenixframework/phoenix/issues/3969 was closed, I should probably remove this change. However, the integration tests won't pass on this version of OTP, so do you mind if the integration tests for elixir 1.9.4 run on a different version of OTP than the unit tests?

Reply:
It is fine, yes. :)